src/test/java/space/obminyashka/items_exchange/controller/ImageControllerTest.java

```diff
@@ -1,13 +1,5 @@
 package space.obminyashka.items_exchange.controller;
 
-import space.obminyashka.items_exchange.exception.ElementsNumberExceedException;
-import space.obminyashka.items_exchange.exception.UnsupportedMediaTypeException;
-import space.obminyashka.items_exchange.model.Advertisement;
-import space.obminyashka.items_exchange.model.Image;
-import space.obminyashka.items_exchange.model.User;
-import space.obminyashka.items_exchange.model.enums.Status;
-import space.obminyashka.items_exchange.service.AdvertisementService;
-import space.obminyashka.items_exchange.service.ImageService;
```

Comment:
You have just replaced the imports and rewrote the commit history.

Reply:
Only because I removed couple classes from here.